### ISSUE TITLE
Contact Alerts

### ISSUE THREAD

**Issue by adamcollison85** [reapit/foundations#10594]
Afternoon,

We are currently pulling in journal entries for our customers.

We have just been made aware of the concept of "Contact Alerts". Can you confirm if these come in via the Journal entries and if so how we target them?

If not how do we access these via the API.

Naturally if an agent has used this feature to flag a customer should not be contacted or if there is information thats important for everyone to see we should also be displaying this at the top of our journal entry feed as it is in RPS.

Any help on this would be appreciated. 

**Specification**
- Project: Reapit.Services.Contacts
- Consider adding new object `alertText` that exposes cnt.cntExtra.cntAlertText and cnt.cntExtra.cntAlertNeg (db also contains an alert modified date)

```
"alert": {
   "negotiatorId": "RPT",
   "text": "Do not contact during UK working hours"
}
```

Linked issue requires write back, so we need to consider this . There are restrictions around who can/cannot update alerts which we will need to understand

**Comment by plittlewood-rpt** [comment]
@neilpellowe for your review

**Comment by plittlewood-rpt** [comment]
Merged into #9807
